src/api/utils.js

```diff
@@ -225,12 +225,13 @@ export const getRestrictedOrgUnits = (orgUnits, orgUnitType) => {
     })
 }
 
-export const appendUsernameToDisplayName = userModelCollection => {
-    userModelCollection.forEach(user => {
-        user.displayName += ` (${user.userCredentials.username})`
+export const appendUsernameToDisplayName = userModelCollection =>
+    userModelCollection.toArray().map(userModel => {
+        const username = userModel.userCredentials.username
```

Comment:
Interestingly `userModel.userCredentials.username` has a value, but `user.userCredentials.username` does not. I suspect this is a bug in the `toJSON` method on the `ModelBase` class in `d2`, but haven't spent much time looking into this, since we're moving away from `d2` anyway.